app/src/main/java/mozilla/lockbox/store/AccountStore.kt

```diff
@@ -224,6 +232,33 @@ open class AccountStore(
 
         this.securePreferences.remove(Constant.Key.firefoxAccount)
         this.generateNewFirefoxAccount()
+        webView.clearCache(true)
+        clearLogs()
+    }
+
+    private fun clearLogs() {
```

Comment:
I'm not sure why these methods are included. We don't write anything sensitive to the logs, so clearing them out when disconnecting seems like extra overhead. Is this related the the webview cache?

Reply:
oops, just re-read that bug. this looks good to me.